### DISCUSSION TITLE
INSTRUCTIONAL VIDEO - How to Configure a Plenti CMS for Editors

### DISCUSSION THREAD

**Discussion by jamestagal** [plentico/plenti#328] [General]
Hi all,

I just released a step-by-step video guide how to set up a Plenti CMS for remote editors.
Hopefully it is useful to others here. Enjoy!

https://www.youtube.com/watch?v=fPfWTYg19Po

Regards,
Ben

**Comment by jimafisk** [comment]
@jamestagal your videos are so good, thank you for creating such a wonderful intro to setting up the Plenti CMS!
